themes/mlibrary_new/custom.php

```diff
@@ -58,11 +58,18 @@ function mlibrary_display_exhibit_type_of_item($rawAttachment)
 
 function mlibrary_exhibit_builder_display_random_featured_exhibit()
 {
-  $exhibits = get_records('Exhibit' , array ('featured'=>true),$num=4);
-  shuffle($exhibits);
-  return $exhibits;
+    $html = '<div id="featured-exhibit">';
+    $featuredExhibit = exhibit_builder_random_featured_exhibit();
+    $html .= '<h2>'.'Feature Exhibits'.'</h2>';
```

Comment:
Make this one string rather than concatenating three short string literals.

Reply:
Or pass 'Featured Exhibits' into a text translation function, whatever Zend uses for that.